src/main/drivers/adc.h

```diff
@@ -24,7 +24,7 @@ typedef enum {
     ADC_CURRENT = 1,
     ADC_EXTERNAL1 = 2,
     ADC_RSSI = 3,
-    ADC_CHANNEL_MAX = ADC_CURRENT
+    ADC_CHANNEL_MAX = ADC_RSSI
```

Comment:
i do not want to change that much. only fix the bug, but yes all those assigments can be removed. only the max one need to be there.

i checked the codebase. its not consistent.
enums like this with counting up and  other enums just start with = 0 and do not count up.

i would even remove the = 0. i think we need some kind of style guide for this stuff(lint) and for formating